resources/sass/components/_copyright_overlay.scss

```diff
@@ -0,0 +1,20 @@
+.copyright-overlay {
+  background-color: rgba(var(--bs-white-rgb),var(--bs-bg-opacity));
+  color: var(--bs-gray-700);
+  padding: 0 0.5rem;
+  display: inline-flex;
+  .preview {
+    display: inline-block;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    max-width: 112px;
```

Comment:
should this be rem so its a relative value to the device viewing?

Reply:
changed it to rem. indeed